botocore/data/pricing/2017-10-15/examples-1.json

```diff
@@ -62,42 +63,6 @@
         "id": "to-retreive-attribute-values",
         "title": "To retrieve a list of attribute values"
       }
-    ],
-    "GetProducts": [
```

Comment:
This is another one where we're completely deleting an operation example. Does this need to be upstreamed?